packages/shared-ux/code_editor/impl/code_editor.stories.tsx

```diff
@@ -199,3 +199,21 @@ export const HoverProvider = () => {
     </div>
   );
 };
+
+export const StaleStateInOnChangeCallback = (params: CodeEditorStorybookParams) => {
+  const [value, setValue] = React.useState('');
+  return (
+    <CodeEditor
+      {...params}
+      languageId="plainText"
+      height={250}
+      onChange={(newValue) => {
+        console.log(`onChange, newValue: ${newValue}, oldValue: ${value}`);
```

Comment:
Here oldValue is always the initial `""` value instead of the previous set value. This might lead to a bunch of bugs depending on how the editor is used